packages/firestore/test/integration/api/validation.test.ts

```diff
@@ -159,7 +158,7 @@ apiDescribe('Validation:', (persistence: boolean) => {
 
     validationIt(persistence, 'garbage collection can be disabled', db => {
       // Verify that this doesn't throw.
-      db.settings({ cacheSizeBytes: CACHE_SIZE_UNLIMITED });
+      db.settings({ cacheSizeBytes: /* CACHE_SIZE_UNLIMITED */ -1 });
```

Comment:
The constant is not exported internally (https://github.com/firebase/firebase-js-sdk/blob/ae319059e4a083a613a2999c1e61d44c9053a0d7/packages/firestore/index.ts)

Reply:
I was able to make it work by importing it from the namespace: `firestore.CACHE_SIZE_UNLIMITED`